setup.py

```diff
@@ -0,0 +1,39 @@
+import os
+from setuptools import setup, find_packages
+
+with open("README.md", "r", encoding="utf-8") as f:
+    LONG_DESCRIPTION = f.read()
+
+# get github workflow env vars
+try:
+    version = (os.environ['GIT_TAG_NAME']).replace('v', '')
+except KeyError:
+    print('Defaulting to dev')
+    version = 'dev'
+
+setup(
+    name='ClashRoyaleBuildABot',
+    version=version,
+    description='A platform for creating bots to play Clash Royale',
```

Comment:
Ensure the descriptions etc are configured properly @Pbatch

Reply:
The GIFs are too large for PyPI (must be <10MB) but everything else looks fine